_posts/0000-01-02-Eoyeyiola.md/1st

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
If you are brave enough to start, you're strong enough to finish!

-- Gary Ryan Blair